charts/calico/templates/calico-node-rbac.yaml

```diff
@@ -176,6 +141,61 @@ rules:
 
 ---
 
+metadata:
+  name: calico-cni-plugin
+rules:
+  # Used for creating service account tokens to be used by the CNI plugin
+  - apiGroups: [""]
+    resources:
+      - serviceaccounts/token
```

Comment:
This permission should only be on the calico-node serviceaccount, not on the CNI plugin